test/tests/php-fpm-hello-web/run.sh

```diff
@@ -8,19 +8,22 @@ image="$1"
 
 # Build a client image with cgi-fcgi for testing
 clientImage='librarytest/php-fpm-hello-web:fcgi'
-docker build -q -t "$clientImage" - > /dev/null <<'EOF'
+docker build -t "$clientImage" - > /dev/null <<'EOF'
```

Comment:
This one is because what `-q` actually does/means upstream isn't super well defined, and we're redirecting the output anyhow.  Just trying to avoid it breaking out from under us if Docker decides to make `-q` mean something.